crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1977,6 +1977,76 @@ impl SymbolicationActor {
             .context("Minidump stackwalk future cancelled")?)
     }
 
+    /// Iteratively stackwalks/processes the given `minidump_file` using breakpad.
+    async fn stackwalk_minidump_iteratively_with_breakpad(
+        &self,
+        scope: Scope,
+        minidump_file: TempPath,
+        sources: Arc<[SourceConfig]>,
+        cfi_caches: &mut CfiCacheModules,
+    ) -> anyhow::Result<StackWalkMinidumpResult> {
+        let mut iterations = 0;
+
+        let result = loop {
+            iterations += 1;
+
+            let result = match self
+                .stackwalk_minidump_with_cfi(&minidump_file, cfi_caches)
+                .await
+            {
+                Ok(res) => res,
+                Err(err) => {
+                    if let Some(dir) = self.diagnostics_cache.cache_dir() {
+                        if let Some(file_name) = minidump_file.file_name() {
+                            let path = dir.join(file_name);
+                            match minidump_file.persist(&path) {
+                                Ok(_) => {
+                                    sentry::configure_scope(|scope| {
```

Comment:
should this be a `with_scope`?  otherwise this crashed minidump stays there for a long time on the scope?

Reply:
At first I thought yes, but looking at the code again, its impossible since this function-scope is completely different from whereever the error itself is being triggered :-(